src/ai/AiModelManager.ts

```
@@ -1,52 +1,76 @@
+                            {
+                                apiKey: globalConfig.models?.agentZero?.apiKey,
+                                endpoint: globalConfig.models?.agentZero?.endpoint,
+                                memory: globalConfig.models?.agentZero?.memory || agentZeroConfig.memory,
+                                toolUsage: globalConfig.models?.agentZero?.toolUsage || agentZeroConfig.toolUsage,
+                                multiAgent: globalConfig.models?.agentZero?.multiAgent || agentZeroConfig.multiAgent,
+                                browserAgent: globalConfig.models?.agentZero?.browserAgent || agentZeroConfig.browserAgent
+                            }
+                        ))
                         break;
                     default:
                         const aiModel = globalConfig.modelFactory?.create(key, editor, globalConfig);
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)